packages/formation/js/back-to-top.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * This button is used for navigating the user back to the top of the page.
+ * It's to be used on long article pages.
+ */
+
+function navigateToTop() {
+  const el = document.querySelector('body.merger');
```

Comment:
Do we need the `.merger`? That class is a vestige of the past (Vets.gov -> VA.gov web brand consolidation) and I'd love to see us kill it with fire at some point.

Reply:
Probably not, but in Storybook there _are_ multiple `<body>` tags. I'll play with this and see what works.

Reply:
Looks like the `<iframe>` limits the scope appropriately, so the `<body>` ends up being the correct one.